test/gpu/native/math.chpl

```diff
@@ -280,7 +317,11 @@ foreach i in r do  R[0] =  j0(r32);   R[1] =  j0(r32);  check(R,"j0(r32)");
 foreach i in r do  R[0] =  j0(r64);   R[1] =  j0(r64);  check(R,"j0(r64)");
 foreach i in r do  R[0] =  j1(r32);   R[1] =  j1(r32);  check(R,"j1(r32)");
 foreach i in r do  R[0] =  j1(r64);   R[1] =  j1(r64);  check(R,"j1(r64)");
-foreach i in r do  R[0] =  jn(2,r32);   R[1] =  jn(2,r32);  check(R,"jn(2,r32)");
+if(!excludeForRocm) {
+  // This will link for ROCM but we get a slightly wrong answer:
+  // > jn(2,r32) computed wrong result. (0.001249, 0.00124896)
```

Comment:
We could try to compile with `--no-ieee-float` to see how results compare (I expect it to change the CPU result, or no change at all.

I also don't mind if we use a custom tolerance for [`isclose`](https://chapel-lang.org/docs/master/modules/standard/AutoMath.html?highlight=isclose#AutoMath.isclose).

Either way, notes for some other day. I prefer merging this PR sooner instead of dwelling on that.